UPGRADE-6.1.md

```diff
@@ -1,6 +1,11 @@
 UPGRADE FROM 6.0 to 6.1
 =======================
 
+All components
+--------------
+
+ * Non-static public and protected properties are now considered final
```

Comment:
```suggestion
 * Non-static public and protected properties are now considered final. Instead of overriding a property, consider setting its value in the constructor.
```